### PR TITLE
Modified Rlink class, dlgOptions dialog and frmEditor

### PR DESCRIPTION
**Pull request by deaspo**
Disabled the autoformat on the grid[still needs a discussion], modified the Rlink to allow different formatting options [still needs discussion to finalize] but not yet connected to the options dialog and added the formating options to the dlgOptions for Comments, R script and R Output. Probably a quick discussion and to finalize all these.

### CODE REVIEW

**Comment by dannyparsons** [comment]
I'll leave this open if you are still working on this.

**Comment by deaspo** [comment]
I have added the sync changes, to finalise the designer for the options dialog

File: instat/clsRLink.vb [hunk 1]
```diff
@@ -30,10 +30,28 @@ Public Class RLink
     Public bClimateObjectExists As Boolean = False
     Public bInstatObjectExists As Boolean = False
     Public bClimsoftLinkExists As Boolean = False
-
+    'sets the font for the strScript
+    Public fScript As Font = New Font(txtOutput.Font.FontFamily, txtOutput.Font.Size)
+    Public clrScript As Color = Color.Black
+    'sets the font for the strOutput
+    Public fOutput As Font = New Font(txtOutput.Font.FontFamily, txtOutput.Font.Size)
+    Public clrOutput As Color = Color.Blue
```
Comment (dannyparsons) [inline instat/clsRLink.vb]: Need to add the same for Comments as well.

File: instat/clsRLink.vb [hunk 2]
```diff
@@ -106,8 +124,9 @@ Public Class RLink
                 txtLog.Text = txtLog.Text & strScript & vbCrLf
             End If
             If bOutput Then
-                txtOutput.Text = txtOutput.Text & strScript & vbCrLf
+
                 'input format here
+                AppendText(txtOutput, clrScript, fScript, strScript & vbCrLf)
```
Comment (dannyparsons) [inline instat/clsRLink.vb]: Also AppendText from comment if not "". And also add to the Log.

File: instat/frmEditor.vb [hunk 3]
```diff
@@ -28,6 +28,10 @@ Public Class frmEditor
         frmMain.clsGrids.SetData(grdData)
         grdData.Visible = False
         autoTranslate(Me)
+        'Disable Autoformat cell
+        'This needs to be added at the part when we are writing data to the grid, not here
+        'Needs discussion, with this the grid can show NA's
+        grdData.SetSettings(unvell.ReoGrid.WorksheetSettings.Edit_AutoFormatCell, False)
```
Comment (dannyparsons) [inline instat/frmEditor.vb]: probably this goes in the GridLink class and is needed for all grids.

File: instat/clsRLink.vb (hunk 1 above)
Comment (deaspo) [inline instat/clsRLink.vb]: Added the formatting for comments also.

**Comment by deaspo** [comment]
Changed the default comment color and added #796

**Pull request merged**